ml3d/datasets/semantickitti.py

```diff
@@ -187,11 +187,12 @@ def save_test_result(self, results, attr):
         make_dir(save_path)
         test_file_name = name_points
 
-        pred = results['predict_labels']
+        pred = results['predict_labels'] + 1
+
+        np.save("pred_label.npy", pred)
 
         store_path = join(save_path, name_points + '.label')
-        for ign in cfg.ignored_label_inds:
-            pred[pred >= ign] += 1
```

Comment:
I guess, this is generalised implementation for multiple ignored indices.

Reply:
Done